tests/drivers/pwm/pwm_to_gpio_loopback/testcase.yaml

```diff
@@ -0,0 +1,57 @@
+common:
+  tags:
+    - pwm
+    - drivers
+    - ci_tests_drivers_pwm
+  depends_on: pwm
+  filter: dt_compat_enabled("test-pwm-to-gpio-loopback")
+  harness: ztest
+  harness_config:
+    fixture: gpio_loopback
+
+tests:
+  drivers.pwm.gpio_loopback:
+    platform_allow:
+      - nrf54h20dk/nrf54h20/cpuapp
+      - nrf54l15dk/nrf54l15/cpuapp
+    integration_platforms:
+      - nrf54h20dk/nrf54h20/cpuapp
+      - nrf54l15dk/nrf54l15/cpuapp
```

Comment:
Only platform allow is needed if contents of integration platform is identical, `integration_platforms` is a subset of `platform_allow`, not the whole set :)